lib/linguist/languages.yml

```diff
@@ -833,7 +833,7 @@ CodeQL:
   ace_mode: text
   language_id: 424259634
   aliases:
-    - ql
```

Comment:
I don't understand; why have you removed the identation spaces? 🤔

Reply:
In fact it wasn't me, but one of the scripts used to add new grammar (in ./script/)